scripts/docs.ts

```diff
@@ -110,20 +110,39 @@ const { values: args } = util.parseArgs({
       type: "string",
       short: "o",
     },
+    help: {
+      type: "boolean",
+      short: "h",
+    },
   },
   allowPositionals: true,
 });
 
-if (!args.path) {
-  console.error(
-    "Usage: docs.ts --path <filepath-or-glob> [--api <api1,api2,...>] [--write] [--output <output-dir>]",
+if (args.help) {
+  console.log("\n");
+  console.log(
+    "Usage: docs.ts [--path <filepath-or-glob>] [--api <api1,api2,...>] [--write] [--output <output-dir>]",
+  );
+  console.log(
+    '  --path, -p    File path or glob pattern to parse (default "packages/**/*.{ts,tsx}")',
   );
-  console.error("  --path, -p    File path or glob pattern to parse");
-  console.error(
+  console.log(
     "  --api, -a     Comma-separated list of specific APIs to generate",
   );
-  console.error("  --write, -w   Write markdown files to output directory");
-  console.error("  --output, -o  Output directory (default: docs/api)");
+  console.log("  --write, -w   Write markdown files to output directory");
+  console.log("  --output, -o  Output directory (default: docs/api)");
+  process.exit(0);
+}
+
+// Resolve file paths using glob patterns
+let pathGlob = args.path || "packages/**/*.{ts,tsx}";
```

Comment:
No longer require `--path` now that we have critical mass - just default to everything